kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/OrderProcessResultDto.java

```diff
@@ -0,0 +1,49 @@
+package com.kodilla.good.patterns.orders.ordering;
+
+public final class OrderProcessResultDto {
+    private final ProductOrder order;
+    private final PlacedProductOrder placedOrder;
+    private final boolean isUserInformed;
+    private final boolean isOrderStored;
+
+    // order can't be null, placed order can.
+    public OrderProcessResultDto(final ProductOrder order,
```

Comment:
If placedOrder contains the same information as order, it does not make sense to store both of them in this result class.